core/trino-web-ui/src/main/resources/webapp/login.html

```diff
@@ -51,7 +51,7 @@
 <body>
 
 <div class="login-form">
```

Comment:
I think we should add `<base href="baseUrl" />` to the document: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base

Reply:
The login form is a static HTML file, so I don't know how we would set a base URL